_dashboards/maps/maps.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Maps
-nav_order: 15
+nav_order: 70
```

Comment:
Need to add a redirect here.

Reply:
Please change the name of this heading to use our naming convention: Present participle phrase + noun-based phrase. For example, Creating maps, Configuring maps, etc.

Reply:
Revised.

Reply:
> Need to add a redirect here.

Added